.github/workflows/node.js.yml

```
@@ -0,0 +1,36 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on: [push]
+  
+  
+
+defaults:
+    run:
+      working-directory: ./webapp/sdf
+  
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+
```

Inside the workflows its recomendet to test your code against different enviroments (or at least its very easy and it wont hurt to do so)...
```suggestion
    runs-on: ${{ matrix.os }}
    strategy:
      matrix:
        os: [windows-latest, ubuntu-20.04]
        node-version: [10.x, 12.x, 14.x]